app/main.py

```diff
@@ -46,12 +46,13 @@
 app.add_exception_handler(ServerError, server_exception_handler)
 app.add_exception_handler(Exception, server_exception_handler)
 
-app.include_router(metrics.router)
-
 
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=settings.cors_origin,
-    allow_credentials=True,
+    allow_origins=["*"],
```

Comment:
Any reason to avoid using the env var `CORS_ORIGIN`? if we dont need to use it any more should we delete it from readme and .env file?

Reply:
Yes sir, what happens is that if I leave it with cors origin it does not recognize the request, it appears as follows:
![image](https://github.com/user-attachments/assets/aef03485-f178-41ac-80b2-12529bb59c1e)

Reply:
I tested this part and found that the settings.cors_origin variable works fine and should be kept, it's necessary to assign it the value of the host from where the requests are made in the .env file